.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+PR description, link to relevant issues
+
+## Testing Steps
+<!-- Steps to test the changes. Remove if not relevant -->
+
+## Checklist
```

Comment:
What is the goal of this checklist? Is it to remind us of what to think about when filing a PR? Or do we want to keep this as record of things that changed? Or do we see this as things that has to be done? Maybe update the heading to reflect what this checklist's purpose is. Ex: "## Did you remember to update these?", "## This PR also affects:"

Reply:
Changed to comments.